src/main/java/com/minecolonies/coremod/network/messages/UpdateChunkCapabilityMessage.java

```diff
@@ -0,0 +1,109 @@
+package com.minecolonies.coremod.network.messages;
+
+import com.minecolonies.api.colony.IColonyTagCapability;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.client.Minecraft;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'net.minecraft.client.Minecraft'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)